client/i18n/locales/english/intro.json

```
@@ -3360,7 +3409,8 @@
       "lab-mood-board": {
         "title": "Build a Mood Board",
         "intro": [
-          "In this lab, you will create a mood board using a React with props."
+          "In this lab, you will create a mood board using React props.",
```

```suggestion
          "In this lab, you will create a mood board using React.",
```
I think it's enough to mention props in the next step.